lib/hanami/view/context_helpers/content_helpers.rb

```diff
@@ -0,0 +1,26 @@
+module Hanami
```

Comment:
@timriley This is fine to put as a proof of concept here, but we should consider to use `hanami-helpers` in the future. 🙂

Reply:
@jodosha haha, I forgot about this package! Yep, I'd be happy to move this work into there when we get to the point of expanding this beyond a single-file experiment.